scripts/tests/calibnet_other_check.sh

```diff
@@ -13,8 +13,8 @@ forest_init
 echo "Verifying the non calibnet snapshot (./test-snapshots/chain4.car) is being served properly."
 $FOREST_CLI_PATH chain read-obj -c bafy2bzacedjrqan2fwfvhfopi64yickki7miiksecglpeiavf7xueytnzevlu
 
-echo "Testing js console"
-$FOREST_CLI_PATH attach --exec 'showPeers()'
+# echo "Testing js console"
+# $FOREST_CLI_PATH attach --exec 'showPeers()'
```

Comment:
Hmm, it is problematic to have features that we are not testing. If we stop testing the JS integration, we should delete it from Forest.

Reply:
Added this back